### DISCUSSION TITLE
add some color props explanation in doc?

### DISCUSSION THREAD

**Discussion by aboveyunhai** [chakra-ui/chakra-ui#3024] [General] [answered]
Just some personal experience regarding the color props of Chakra-UI.
I am currently having some annoying experiences when try to add color into Chakra UI component.

For example, `bgColor`, `backgroundColor` , `color` and `colorScheme`, etc.
Some of them do accept the format `teal.100` but some does not. Some accept CSS style like `antiquewhite`, `black` etc. vice versa. Also similar names like `teal` and `teal.100` can cause further confusion.
I never find a clear way to sort out those props especially when some of components accept multiple color props, and had to consistently check back the doc. Really wish there is a better way to clarify those standards.

**Comment by chr-ge** [comment, marked as answer]
All color props (`bg`, `color`, `borderColor`, etc) except `colorScheme` accept css color values and palette values (`blue.100`).
>The colorScheme prop represents a visual color scheme, not a css color attribute.

You give the `colorScheme` a color name such as "blue", "green", "red", etc. You can pass it any of the default theme colors. For example, If you give a Button component a colorScheme of "red", it uses the different palette ranges for its color and hover, disabled, active pseudo-classes.

**Reply by with-heart** [reply]
Solid answer @chr-ge!

Also are you in our Discord by any chance?

**Reply by chr-ge** [reply]
No, I'll join now.

**Comment by with-heart** [comment]
@aboveyunhai We should definitely add more content around this to the docs. Thanks for the suggestion!
